services/core/config.js

```diff
@@ -133,4 +136,11 @@ config.queue = {
  */
 config.jobs = {};
 
+config.snapshot = {
+	url: process.env.INDEX_SNAPSHOT_URL,
+	composeFilePath: process.env.DOCKER_COMPOSE_FILEPATH,
+	serviceName: process.env.DOCKER_SERVICE_NAME,
+	enable: Boolean(String(process.env.ENABLE_APPLY_SNAPSHOT).toLowerCase() !== 'false'), // Enable by default
```

Comment:
I think snapshot application should be disabled it by default.
```suggestion
	enable: Boolean(String(process.env.ENABLE_APPLY_SNAPSHOT).toLowerCase() === 'true'), // Disabled by default
```